docs/docsite/rst/community/other_tools_and_programs.rst

```diff
@@ -99,7 +99,7 @@ Other tools
 ===========
 
 - `Ansible Inventory Grapher <https://github.com/willthames/ansible-inventory-grapher>`_ - visually displays inventory inheritance hierarchies and at what level a variable is defined in inventory.
-- `Ansible Shell <https://github.com/dominis/ansible-shell>`_ - an interactive shell for Ansible with built-in tab completion for all the modules.
+- `Ansible Shell <https://github.com/dominis/ansible-shell>`_ - included with the Ansible distribution as `ansible-console` and provides an interactive shell with built-in tab completion for all modules.
```

Comment:
So this is a deprecated project that's now been rolled into ansible-core, correct? In that case, we should just completely remove the reference.

```suggestion
```